test/card-test.js

```diff
@@ -122,16 +121,17 @@ describe('cards interactions', function () {
     assert.notEqual(card.lastInteraction, lastInteractionBefore, msg3);
   });
 
-  // it('card dragability can be disabled', function () {
-  //   let card = new Card();
-  //   $(card.card).data('draggable');
-  //   if ($(card.card).draggable('disable'))
-  //     var disabled = 1;
-  //   return assert.equal(disabled, 1);
-  // });
+  it('card dragability can be disabled', function () {
```

Comment:
Testing whether a local variable (`disabled`) equals 1 after evaluating an `if` statement which limits that variable from being declared only when a `Card` class instance allows the `jquery-ui.draggable('disable')` command is a masked testing methodology. Since the ability to disable draggability is inherited from the `jquery-ui` package, this test is actually evaluating the functionality of that package. Tests need to focus on evaluating the functionality of the `synectic` app.